interface.go

```diff
@@ -88,6 +88,8 @@ type Config struct {
 	// MaxReceiveConnectionFlowControlWindow is the connection-level flow control window for receiving data.
 	// If this value is zero, it will default to 1.5 MB for the server and 15 MB for the client.
 	MaxReceiveConnectionFlowControlWindow protocol.ByteCount
+	// Keepalive defines whether keep-alive is enabled.
+	Keepalive bool
```

Comment:
Generally it is referred to as `Keepalive`.
I referred to the Linux kernel docs [here](http://www.tldp.org/HOWTO/html_single/TCP-Keepalive-HOWTO/) to check if it should be `KeepAlive` or `Keepalive`.

Let me know if I should stick with it or make it `KeepAlive`.